dask/array/tests/test_cupy_creation.py

```diff
@@ -158,7 +158,7 @@ def test_pad(shape, chunks, pad_width, mode, kwargs):
     ],
 )
 def test_tri_like(xp, N, M, k, dtype, chunks):
-    xp_tri = getattr(xp, "tri")
+    xp_tri = xp.tri
```

Comment:
Since `xp_tri` is only ever used in one line, let's just inline it directly below. So instead of
```python
xp_a = xp_tri(*args, like=da.from_array(cupy.array(())))
```

let's have

```python
xp_a = xp.tri(*args, like=da.from_array(cupy.array(())))
```

This will reduce the number of variables code readers need to keep in their head when looking through this test

Reply:
Good suggestion! I've added it in.